layouts/shortcodes/youtube-width.html

```diff
@@ -0,0 +1,11 @@
+{{- $pc := .Page.Site.Config.Privacy.YouTube -}}
```

Comment:
I disagree and think we should merge this. It's not likely there will be any additional options needed. Even if additional options are added/available, the primary use of this extension will always be to limit the width of embedded YouTube videos.

Reply:
I see your point on the shortcode being a special purpose optimization that's only intended to limit the width. It's just that the way the shortcode is written, it's designed to be general purpose. It still takes `width` as a parameter, for example. We've just changed the default to be 50%.

I could disagree and commit though if just the documentation in the Contributor Guide were updated to state that this shortcode purposely sets the default to 50% (i.e. the primary use case, as you stated). The example should then be updated to show this typical use case, where width is not used. Optionally, an additional code block could show what the shortcode looks like when it is used.

Reply:
For the current purpose of the shortcode, I think `youtube-width` is fitting. (If it were to change with additional customization, the entire shortcode would change and a renaming would be fitting.)

I see your point @willihay, since the shortcode is essentially the built-in `youtube` shortcode but with an extension: `width`. If there were more customization options for a video shortcode that we need, `youtube-ext` makes sense. But I agree with @micronAMZN that it's not likely there's much else to add. I also lean towards `youtube-width` because it follows the same naming convention as `image-width` and is easier to remember.

Reply:
OK. What about the suggestion in my last comment about the updates that I recommend for the contributor guide example? It's not clear to me if you plan on taking that suggestion. If you would, click the "re-request review" button so I know when you're done. Thanks!

Reply:
Yes I'll adjust the examples to show how to use `youtube-width` with the `width` parameter and without.

Reply:
@willihay Here is a preview of the examples. I believe this is what you mean? 
![image](https://user-images.githubusercontent.com/75444793/146269707-f4b7466a-8958-42b8-a2b7-bc6d25250551.png)